_content/essays/japanese-buddhist-world-map_moerman-david-max.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "The Japanese Buddhist World Map: Religious Vision and the Cartographic Imagination"
+authors:
+  - "David Max Moerman"
+editor: "Shatrunjay Mall"
+external_url: "https://newbooksnetwork.com/the-japanese-buddhist-world-map"
+course: japanese
+tags:
+  - cosmology
+  - bart
```

Comment:
bart is fine here. Can always move down later if needed.

The "cartography" tag is [`maps`](https://buddhistuniversity.net/tags/maps) 😊